TicketingContract.sol

```diff
@@ -238,9 +314,9 @@ contract TicketPro
         selfdestruct(organiser);
     }
 
-    function isStormBirdContract() public pure returns (bool) 
+    function isStormBirdContract() public pure returns (bool)
     {
-        return true; 
+        return true;
     }
 
```

Comment:
Need to update this to return which interfaces are available so the client can interact with the contract correctly.